src/RestControllers/PatientRestController.php

```diff
@@ -55,23 +55,23 @@ function post($data)
 
     /**
      * Processes a HTTP PUT request used to update an existing patient record.
-     * @param $pid - The patient identifier used to lookup the existing record.
+     * @param $puuidString - The patient uuid identifier in string format.
      * @param $data - array of patient fields (full resource).
      * @return a 200/Ok status code and the patient resource.
      */
-    function put($pid, $data)
+    function put($puuidString, $data)
     {
-        $processingResult = $this->patientService->update(intval($pid), $data);
+        $processingResult = $this->patientService->update($puuidString, $data);
         return RestControllerHelper::handleProcessingResult($processingResult, 200);
     }
 
     /**
      * Fetches a single patient resource by id.
-     * @param $pid The patient identifier to fetch.
+     * @param $puuidString - The patient uuid identifier in string format.
      */
-    function getOne($pid)
+    function getOne($puuidString)
     {
-        $processingResult = $this->patientService->getOne(intval($pid));
+        $processingResult = $this->patientService->getOne($puuidString);
```

Comment:
So I was trying to do something similar but as we have ended the war here by using only UUIDs.
So what should we do with encounters and other resources?
Should we support lookups with both UUIDs and EID or just UUID? [Regarding Standard API]
And If both how to check whether the ID is UUID or EID?

Reply:
Just gonna be uuids for here on out for the api. It is a brave new world :)
(i played around with supporting both for patient endpoints since uuid has `-`, but this just ended up making code way to wonky; so full steam ahead on uuid)